src/current/v23.2/migration-strategy-live-migration.md

```diff
@@ -0,0 +1,100 @@
+
+Advantages:
+
+- There's greater service availability with live migrations. You can failback to the source database at any point in the migration process, or if there are significant errors after cutover.
+- The impact of a migration on users is minimized by having little to no downtime.
+- There's less internal coordination cost. For example, if other teams use the database, they can update their systems during the initial phase of the migration with no extended outage or maintenance window.
+- If you use dual-writes, you can test the viability and scalability of your architecture on live production data. For example, you can tune the performance of CockroachDB before the cutover without affecting the performance of the production application.
+
+Disadvantages:
+
+- It is complex to implement. You need the right application design and replication tools to successfully perform the migration.
+- It requires abstracting the database operations in the application. You need complete control over all the applications reading and writing to the database to ensure the data is consistent across both the source and target database.
+- During migration there's less certainty about the integrity of the data due to multiple systems being updated.
+- Hidden dependencies may reveal themselves after cutover. For example, a microservice that was not accounted for during the migration planning and uses the source database may break after cutover.
+- The operational cost of two production systems can be expensive.
+
+## Prepare for a live migration
+
+Before performing a live migration, consider the following:
+
+- It is better to over-provision your cluster during the migration and then scale it back after the migration is complete than to discover you need more cluster resources during the migration.
+- Testing against the live production workload by shadowing traffic from the source database to CockroachDB allows you to properly size your cluster and increases the likelihood that your migration will succeed with minimal problems
+- Writes are typically where performance problems arise during live migration. Testing your workload beforehand can help identify areas with slow write performance so they can be fixed before starting the migration.
+- After the migration completes, you should factor in time for performance tuning.
+- A common way of testing the performance of your workload on CockroachDB is to do gradual testing while the source database is the source of truth, and the CockroachDB cluster's data is replicated from the source database.
+- Fallback is typically quite expensive, and it is usually easier to fix any data consistency issues after the cutover to the target database than to fallback to the original source database.
+
+## Live migration tooling
```

Comment:
Should the LMS be included in this section? I think the main points are: verifying application queries to source; handling cutover to target; verifying application queries on target.